silver/utils/pdf.py

```diff
@@ -9,25 +9,26 @@ class UnsupportedMediaPathException(Exception):
 
 def fetch_resources(uri, rel):
     """
-    Callback to allow xhtml2pdf/reportlab to retrieve Images,Stylesheets, etc.
-    `uri` is the href attribute from the html link element.
-    `rel` gives a relative path, but it's not used here.
+    Convert HTML URIs to absolute system paths so xhtml2pdf can access those
+    resources
     """
-    if settings.MEDIA_URL and uri.startswith(settings.MEDIA_URL):
-        path = os.path.join(settings.MEDIA_ROOT,
-                            uri.replace(settings.MEDIA_URL, ""))
-    elif settings.STATIC_URL and uri.startswith(settings.STATIC_URL):
-        path = os.path.join(settings.STATIC_ROOT,
-                            uri.replace(settings.STATIC_URL, ""))
-        if not os.path.exists(path):
-            for d in settings.STATICFILES_DIRS:
-                path = os.path.join(d, uri.replace(settings.STATIC_URL, ""))
-                if os.path.exists(path):
-                    break
-    elif uri.startswith("http://") or uri.startswith("https://"):
-        path = uri
+    # use short variable names
+    sUrl = settings.STATIC_URL      # Typically /static/
```

Comment:
Please use the usual python variable styling: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
Writing
```python
static_url = settings.STATIC_URL
static_root = settings.STATIC_ROOT
...
```
is totally fine in Python.